back/src/program/repositories/in-memory-program.repository.ts

```diff
@@ -2,23 +2,27 @@ import { ProgramRepository } from './program-repository.interface'
 import { CreateProgramInput } from '../types/create-program-input.type'
 import { Program } from '../entities/program.entity'
 import { v4 as uuid } from 'uuid'
-import { programDataBuilder } from '../data-builders/program.data-builder'
+import { programFixtures } from '../data-builders/program.data-builder'
 import { UpdateResult } from 'typeorm'
+import { Workout } from '../../workout/entities/workout.entity'
 
 export class InMemoryProgramRepository implements ProgramRepository {
-  private programsData = [
-    programDataBuilder(),
-    programDataBuilder(),
-    programDataBuilder(),
-    programDataBuilder(),
-  ]
+  private programsData = programFixtures
 
   private programs = this.programsData.map((data) => new Program(data))
```

Comment:
Fixtures cannot be used for InMemory, Put dataBuilder again.

InMemory is for Unit test, decoupled from the DB.

Imagine we use MongoDb ? This InMemory, using SQL fixtures, will break.